recipe/meta.yaml

```
@@ -21,29 +23,47 @@ requirements:
   run:
     - python
     - six
-    - scandir  # [py27]
+    - scandir  # [py<35]
+    - typing  # [py<35]
```

```suggestion
  run_constrained:
    - scandir  # [py<35]
    - typing  # [py<35]
```
Move them to `run_constrained` because they are optional dependencies https://github.com/jazzband/pathlib2/blob/2.3.7-post1/setup.py#L45-L47

---

Sergii I would not do that, 
if user install this pkg under py<35 conda install only pathlib2 but not scandir, typing and code will break
this should be in **run** but with selectors accroding to rules of upsteram pkg-info